app/assets/stylesheets/application.scss

```diff
@@ -15,10 +15,48 @@
  *= require foundation_and_overrides
  */
 
+@import url(https://fonts.googleapis.com/css?family=Open+Sans:400,300,600,700,400italic,300italic);
+
  /************** BASIC STYLING *****************************/
```

Comment:
I would factor out the styles in the file into their own CSS file. It's usually best to avoid adding styles or JS to the application manifest file.